google_compute_engine/accounts/accounts_utils.py

```diff
@@ -43,8 +44,8 @@ def __init__(self, logger, groups=None, remove=False):
     """
     self.logger = logger
     self.google_sudoers_group = 'google-sudoers'
-    self.google_sudoers_file = '/etc/sudoers.d/google_sudoers'
-    self.google_users_dir = '/var/lib/google'
+    self.google_sudoers_file = constants.LOCALBASE + '/etc/sudoers.d/google_sudoers'
```

Comment:
Please keep line lengths under 80 characters. I'd suggest updating this with:

```
self.google_sudoers_file = (
    constants.LOCALBASE + '/etc/sudoers.d/google_sudoers')
````